Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore.py

```diff
@@ -30,16 +30,26 @@
 for entry in data:
     indicator_type = entry["indicator_type"]
     score = entry["score"]
-    source = entry.get('source', '')
+    sources = entry.get('sourceBrands', [])
     if hasattr(INDICATOR_TYPE_DICTIONARY, indicator_type):
         indicator_type = INDICATOR_TYPE_DICTIONARY[indicator_type]
 
-    ec["DBotScore"].append({
-        "Indicator": indicator,
-        "Type": indicator_type,
-        "Vendor": source,
-        "Score": score
-    })
+    if sources:
+        for source in sources:
+            ec["DBotScore"].append({
+                "Indicator": indicator,
+                "Type": indicator_type,
+                "Vendor": source,
+                "Score": score
+            })
+    else:
+        ec["DBotScore"].append({
+            "Indicator": indicator,
+            "Type": indicator_type,
+            "Vendor": '',
+            "Score": score
+        })
```

Comment:
why would it happen?

Reply:
Im not sure why but its possible situation, i had some in my local env:
![image](https://user-images.githubusercontent.com/46249224/93449111-06935080-f8dd-11ea-86a7-c7273a5327a3.png)